### PR TITLE
Parameters utils error

### PR DESCRIPTION
**Pull request by huangyiminghappy**
documentContext.read may occur exception,so i think we need to catch the exception.

### CODE REVIEW

File: core/src/main/java/com/netflix/conductor/core/execution/ParametersUtils.java [hunk 1]
```diff
@@ -214,7 +213,11 @@ private Object replaceVariables(String paramString, DocumentContext documentCont
                     }
 
                 } else {
-                    convertedValues[i] = documentContext.read(paramPath);
+                    try {
+                        convertedValues[i] = documentContext.read(paramPath);
+                    } catch (Exception e) {
```
Comment (cyzhao) [inline core/src/main/java/com/netflix/conductor/core/execution/ParametersUtils.java]: Add warning level logging

**Comment by apanicker-nflx** [comment]
Please resolve merge conflicts and add tests.

File: core/src/main/java/com/netflix/conductor/core/execution/ParametersUtils.java (hunk 1 above)
Comment (huangyiminghappy) [reply core/src/main/java/com/netflix/conductor/core/execution/ParametersUtils.java]: thank you review, please review the https://github.com/Netflix/conductor/pull/988 , if it ok ,current pr will close

**Comment by huangyiminghappy** [comment]
> Please resolve merge conflicts and add tests.

thank you review, please review the #988 , if it ok ,current pr will close

**Comment by kishorebanala** [comment]
Reopened with #988

**Pull request closed without merging**